crates/solver/src/settlement.rs

```diff
@@ -398,23 +399,38 @@ impl Settlement {
                     Some(price) => price,
                     None => return true,
                 };
-                // Condition to check: Deviation of clearing prices is bigger than max_settlement_price deviation
+
+                // Condition to check: Deviation of clearing prices is bigger than
+                // max_settlement_price deviation
+                //
+                // clearing_price_sell_token / clearing_price_buy_token -
+                // external_price_sell_token / external_price_buy_token
+                // ------------------------------------------------------
+                // clearing_price_sell_token / clearing_price_buy_token
+                //
+                // is bigger than
                 //
-                // |clearing_price_sell_token / clearing_price_buy_token - external_price_sell_token / external_price_buy_token)|
-                // |----------------------------------------------------------------------------------------------------------|
-                // |                     clearing_price_sell_token / clearing_price_buy_token                                 |
-                // is bigger than:
                 // max_settlement_price_deviation
                 //
-                // This is equal to: |clearing_price_sell_token * external_price_buy_token - external_price_sell_token * clearing_price_buy_token|>
-                // max_settlement_price_deviation * clearing_price_buy_token * external_price_buy_token * clearing_price_sell_token
+                // This is equal to:
+                //
+                // clearing_price_sell_token * external_price_buy_token -
+                // external_price_sell_token * clearing_price_buy_token
```

Comment:
The original comment took the absolute value of this.

Reply:
fixed